dev-docs/source/Testing/MuonAnalysis_test_guides/Muon Analysis Unscripted Testing Guide.rst

```diff
@@ -8,46 +8,85 @@ Muon Interfaces Unscripted Testing
 
 Preamble
 ^^^^^^^^^
-This document is intended for developers to use for unscripted testing of `Muon Analysis` and
-`Frequency Domain Analysis` GUI's.
+This document is intended for developers to use for unscripted testing of muon GUI's.
 
 The tests follow real use cases provided by scientists and are intended to exercise all the interface's functionality.
 As changes are made to the interface and features added, anything for which it is not possible to write an automated
 test should have a manual test added to this list.
 
-.. note:: The tests here are grouped into four sections. The test groups should be run in order and, within each test
-   group, the individual tests should also be run in order. This is because, to avoid duplication, some tests rely on
-   previous tests having been run first.
+.. note:: The tests here are grouped into sections. The test groups can be done in any order.
+
 
 Common setup
 ^^^^^^^^^^^^
 - Set your facility to ISIS
-- Ensure the files ``EMU00020918-20``, ``MUSR00015189`` and ``MUSR00062260-1`` are in Mantid's path
-- Open :menuselection:`Interfaces --> Muon --> Muon Analysis`
-- At this point, before loading any data, the *Grouping Options* and *Data Analysis* tabs should be disabled.
+- Ensure you can acces the files for the test (listed below).
```

Comment:
Where can these files be found, in the ISIS Training data, or Sample Data?

Reply:
Its in none of them. This is from the old testing guide https://developer.mantidproject.org/Testing/MuonInterface/MuonTesting.html 
These are based on real user experiments. So we are not able to put the data in a public place. However, its the only data we have been given that can be used to test the functionality. The same is true for the other tests